testsuite/gnat2goto/tests/arrays_attributes/arrays_attributes.adb

```diff
@@ -0,0 +1,17 @@
+procedure Arrays_Attributes is
```

Comment:
would be helpful to list the attributes at the top in a comment so people know at a glance what to look for:
e.g 
--  array attributes are: first, last, range, length

Reply:
Done.